html-forms/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>html-forms</title>
+  </head>
+  <body>
+    <h1> Coding Form!</h1>
+    <form>
+      <div class="section">
+        Email
+        <label>
+          <input type="text" name="email"/>
+        </label>
+      </div>
+      <div class="section">
+        Why is Coding cool?<br>
+        <select name="dropdown">
+          <option value="select an option">Select an option</option>
+          <option value="because it's interesting">Because it's interesting</option>
+          <option value="because it's fun">Because it's fun</option>
+          <option value="because it's powerful">Because it's powerful</option>
+        </select>
```

Comment:
This `select` needs a `label`. I would put the text on line 16, inside the `label` and link the two using a `for` attribute